draft-pda-protocol.md

```diff
@@ -419,12 +418,15 @@ Duration uint64; /* Number of seconds elapsed between two instants */
 Time uint64; /* seconds elapsed since start of UNIX epoch */
 ~~~
 
-* `uuid`: A unique sequence of bytes used  to ensure that two otherwise
+* `uuid`: A unique sequence of bytes used to ensure that two otherwise
   identical `Param` instances will have distinct `TaskID`s. It is
   RECOMMENDED that this be set to a random 16-byte string derived from a
   cryptographically secure pseudorandom number generator.
-* `leader_url`: The leader's endpoint URL.
-* `helper_url`: The helper's endpoint URL.
+* `aggregator_endpoints`: Vector of URLs relative to which an aggregator's API
+  endpoints can be found. The leader's endpoint MUST be the first in the vector.
+  The order in which aggregators appear in this vector MUST be consistent with
+  the order of the encrypted input shares in a `Report` (see
+  {{uploading-reports}}).
```

Comment:
I would rewrite this because it's the reports which have to be consistent with this:
```suggestion
  endpoints can be found. The leader's endpoint MUST be the first in the vector.
  The order in encrypted input shares in a `Report` (see
  {{uploading-reports}}) MUST be the same as the
  order in which aggregators appear in this vector.
```